app/views/content/landing/grow/_collage.html.erb

```diff
@@ -7,7 +7,8 @@
           checklist: [
             "career progression ",
             "personal development",
-            "opportunities to increase your pay"
+            "opportunities to increase your pay",
+            "a starting salary of $salaries_starting_min$ (or higher in London)",
```

Comment:
@Sarah-DfE I agree with your suggestion - these two bullets would probably make more sense the other way round, so salary is third and increase pay is fourth